index.html

```diff
@@ -288,15 +288,16 @@ <h3 class="titleA">About My self</h3>
             <h2>Contact me</h2>
             <p>If you have an application you are 
               interested in developing, a feature that you 
-              need built or a project that needs coding. I’d love 
+              need built or a project that needs coding.<br> I’d love 
               to help with it!</p>
+      </div>
 
 
-          <form action="https://formspree.io/f/mjvqvayk" method="post">
+          <form action="https://formspree.io/f/xpzgwrza" method="post">
```

Comment:
Just as the previous reviewer requested, your form is still not properly integrated with the formspre service😉. the error below is still displayed when you try to submit. if properly integrated, it should show a prompt that the action was successful. kindly ensure it is properly integrated before submitting, so you don't have to waste reviews and cause blockers for yourself😉.
<img width="780" alt="image" src="https://github.com/sadat-Ramin/My_first_portfolio/assets/64643995/f9f5d964-5424-49f1-81f7-f9436950aad8">

carefully follow the[ steps here ](https://formspree.io/html/)and make your changes👍🏼

Reply:
but I cant make a link like this in formspree